composer.json

```diff
@@ -16,17 +16,17 @@
         }
     ],
     "require": {
-        "php": "^7.0",
-        "illuminate/support": "^5.4.30",
+        "php": "^7.1",
+        "illuminate/support": "^6.0",
         "league/flysystem": "^1.0.41",
         "spatie/db-dumper": "^2.11",
-        "spatie/laravel-migrate-fresh": "^1.4.1",
+        "spatie/laravel-migrate-fresh": "dev-ft-laravel-6",
```

Comment:
I'll probably not merge anything that uses dev branches.

Reply:
This is right! just had to do it to have an installable version. The open todos are listed in the initial comment.